src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig

```diff
@@ -23,31 +23,4 @@
             verticalLayout: data.verticalLayout
         }
     } only %}
-        {% block body %}
-            {% if data.title is not empty %}
-                <div class="col col--sm-12">
-                    <h2 class="{{ config.name }}__title title title--h5 spacing-bottom spacing-bottom--big {{ not data.verticalLayout ? config.name ~ '__title--row' }}">
-                        {{ data.title }}
-                    </h2>
-                </div>
-            {% endif %}
```

Comment:
What about the design changes, has it been approved by the PO?

Reply:
This part was missed during integration from `b2b-demo-shop-internal`
https://github.com/spryker/b2b-demo-shop-internal/pull/1509/files#diff-eefa27467167226909d1dc6ae87b1bd6b836d09286bfc9075e98e8782e40dc3a

Reply:
> This part was missed during integration from `b2b-demo-shop-internal` https://github.com/spryker/b2b-demo-shop-internal/pull/1509/files#diff-eefa27467167226909d1dc6ae87b1bd6b836d09286bfc9075e98e8782e40dc3a

It doesn't mean that title and spaces should be changed during integration.

Reply:
All changes were done in the `product-quick-add-form.twig`, so this embed is redundant

Reply:
Didn't notice that the `product-quick-add-form` was extended at the project level.